### DISCUSSION TITLE
`dependencies` table in data dump

### DISCUSSION THREAD

**Discussion by nasifimtiazohi** [rust-lang/crates.io#4793] [Q&A]
I am struggling to understand the `dependencies` table in the experimental database dump. I assumed the table would contain the list of direct dependencies of a crates as can be found on the crates.io website. But neither the table contains a link to the dependency crates, nor the `version_id` matches with the `id` in the `versions` table.

**Comment by Turbo87** [comment]
There `version_id` points to the version that has the dependency in its `Cargo.toml` file. Each row should also contain the crate name and the semver requirement of the depended upon crate.
